shell/platform/darwin/ios/framework/Source/FlutterViewController.mm

```diff
@@ -25,6 +25,9 @@
 
 NSNotificationName const FlutterSemanticsUpdateNotification = @"FlutterSemanticsUpdate";
 
+// This is left a FlutterBinaryMessenger privately for now to give people a chance to notice the
+// change.  Unfortunately unless you have Werror turned on, incompatible pointers as arguments
+// are just a warning.
 @interface FlutterViewController () <FlutterBinaryMessenger>
```

Comment:
Can we leave a bug to remove this at some point?

Reply:
https://github.com/flutter/flutter/issues/35945 done